v2/rscg_examples_site/docs/indexRSCG.md

```
@@ -20,6 +20,7 @@ This is the list of 288 ( 16 from Microsoft) RSCG with examples
 
 | No        | Name  | Date | Category |
 | --------- | ----- | ---- | -------- |  
+|289| [AOP.Logging.SourceGenerator by Dietmar Borgards ](/docs/AOP.Logging.SourceGenerator)|2026-08-30 => 30 August 2026 | [EnhancementClass](/docs/Categories/EnhancementClass) |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)